cygnus-common/src/main/java/com/telefonica/iot/cygnus/nodes/CygnusApplication.java

```diff
@@ -162,6 +159,14 @@ public synchronized void handleConfigurationEvent(MaterializedConfiguration conf
      */
     public static void main(String[] args) {
         try {
+            // Set some MDC logging fields to 'N/A' for this thread
+            // Later in this method the component field will be given a value
+            org.apache.log4j.MDC.put(CommonConstants.LOG4J_CORR, "N/A");
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1427/commits/8e6687fc9825b8ba22b7cd4d81a61fdda77134a5